spec/requests/waste_exemptions_engine/back_office_edit_complete_forms_spec.rb

```diff
@@ -17,21 +17,23 @@ module WasteExemptionsEngine
       it "renders the appropriate template, returns a 200 status code and W3C valid HTML content", :vcr do
         get request_path
 
-        expect(response).to render_template("waste_exemptions_engine/back_office_edit_complete_forms/new")
-        expect(response).to have_http_status(:ok)
-        expect(response.body).to have_valid_html
+        aggregate_failures do
+          expect(response).to render_template("waste_exemptions_engine/back_office_edit_complete_forms/new")
+          expect(response).to have_http_status(:ok)
+          expect(response.body).to have_valid_html
+        end
       end
 
       context "when the host application has a current_user" do
         let(:current_user) { double }
 
-        # rubocop:disable RSpec/AnyInstance
         before do
           allow(current_user).to receive(:id).and_return(1)
           allow(WasteExemptionsEngine.configuration).to receive(:use_current_user_for_whodunnit).and_return(true)
+          # rubocop:disable RSpec/AnyInstance
```

Comment:
You've done a huge job of cleaning up most of the inline rubocop exceptions. Can this one be also removed by refactoring code?

I don't have a clear solution, but maybe stubbing current_user would work?
ApplicationController.stub(:current_user).and_return(current_user)

If not, then we could try mocking BackOfficeEditCompleteFormsController controller
let(:controller_instance) { instance_double(BackOfficeEditCompleteFormsController) }
allow(controller_instance).to receive(:current_user).and_return(current_user)